src/strings/pt-br.json

```diff
@@ -859,6 +859,7 @@
     "OptionDatePlayed": "Data de Reprodução",
     "OptionDisableUser": "Desativar este usuário",
     "OptionDisableUserHelp": "O servidor não permitirá nenhuma conexão deste usuário. Conexões existentes serão encerradas imediatamente.",
+    "OptionEnableNoPasswordHelp": "Permitirá que este usuário defina sua senha como vazia ao alterar senhas.",
```

Comment:
I believe you should only edit en-us.json as the other files are handled by weblate.